vector/src/main/java/im/vector/activity/LoginActivity.java

```diff
@@ -339,6 +340,8 @@
     // use to reset the password when the user click on the email validation
     private ThreePidCredentials mForgotPid = null;
 
+    Boolean shouldAutodiscoverDomainBasedOnMail = false;
```

Comment:
I would remove this code and all related to this.

Reply:
ok